### PR TITLE
add LLM Log Transfer scope

### PR DESCRIPTION
**Pull request by MadDataScience**
## Related Issues

- T-40032

## Security Implications

_[none]_

## System Availability

_[none]_

### CODE REVIEW

File: src/scopes.ts [hunk 1]
```diff
@@ -986,6 +987,13 @@ const SCOPES_WITHOUT_VIEW_ONLY: {
     type: ScopeType.Modify,
     products: [TranscendProduct.PreferenceManagement],
   },
+  [ScopeName.LLMLogTransfer]: {
+    title: 'LLM Log Transfer',
+    dependencies: [ScopeName.ManageDataInventory],
+    description: 'Ability to transfer logs from LLM to Transcend',
+    type: ScopeType.Modify,
+    products: [TranscendProduct.StructuredDiscovery],
+  }
```
Comment (dipack95) [inline src/scopes.ts:996]: Not sure what we had in mind here, but it might make sense to just have a single, general purpose logging API key? Either way, this is good!

File: .pnp.cjs
Comment (dipack95) [inline .pnp.cjs:1]: Lol why is this needed?

**Review by dipack95 — COMMENTED** [review]
The PR looks good, but the pnp.cjs changes do not -- can we revert them? @MadDataScience

File: .pnp.cjs
Comment (MadDataScience) [reply .pnp.cjs:1]: I have no idea. See https://github.com/transcend-io/privacy-types/actions/runs/11710341566/job/32616413149?pr=188

Comment (dipack95) [reply .pnp.cjs:1]: That's extremely sus, I don't know why it's doing that

File: src/scopes.ts (hunk 1 above)
Comment (michaelfarrell76) [reply src/scopes.ts:996]: i think we should do it like this for now. 

we don't want to add an api key to sombra, if you check height ticket,  ideal future state is some sort of reverse proxy where we would remove this entirely

**Review by michaelfarrell76 — APPROVED** [review]

File: src/scopes.ts [hunk 2]
```diff
@@ -986,6 +987,13 @@ const SCOPES_WITHOUT_VIEW_ONLY: {
     type: ScopeType.Modify,
     products: [TranscendProduct.PreferenceManagement],
   },
+  [ScopeName.LLMLogTransfer]: {
+    title: 'LLM Log Transfer',
+    dependencies: [ScopeName.ManageDataInventory],
+    description: 'Ability to transfer logs from LLM to Transcend',
+    type: ScopeType.Modify,
+    products: [TranscendProduct.StructuredDiscovery],
```
[the remark this answers is not in the document]
Comment (MadDataScience) [reply src/scopes.ts:995]: I was thinking about that. Will do

**Comment by michaelfarrell76** [comment]
@MadDataScience not sure whats up with your environment and the pnp file but id just revert that pnp change to whatever is on base branch

**Comment by MadDataScience** [comment]
> @MadDataScience not sure whats up with your environment and the pnp file but id just revert that pnp change to whatever is on base branch

To be clear, I did not originally touch the pnp file. When I created the PR, [run-pre-commits](https://github.com/transcend-io/privacy-types/actions/runs/11710341566/job/32616413149?pr=188#logs) failed. So I ran `yarn` and it made these changes to the pnp file. If I checkout `main` and run `yarn` there, it makes the same change, so it doesn't have to do with my PR. I don't think it has to do with my environment either, because it's the same in CI. I just reverted the change and...it seems okay this time? I have no idea.

**Comment by michaelfarrell76** [comment]
@MadDataScience ya that makes sense, i have the same issues here - i think it's something that needs to be fixed with a codespace config or by changing ci/not committing pnp file or some other config change that's not that important to fix right now since most of these changes can be fixed without needing the environment to be perfect

**Review by michaelfarrell76 — APPROVED** [review]
